### PR TITLE
JWWEB-399 Make Ethereum asset always active

### PR DESCRIPTION
**Pull request by Dragonesis**

### CODE REVIEW

File: src/components/AssetItem/AssetItem.js [hunk 1]
```diff
@@ -167,6 +168,7 @@ class AssetItem extends PureComponent<Props, StateProps> {
                 <JSwitch
                   onChange={setIsActive}
                   isChecked={isActive}
+                  isDisabled={isActive ? checkETH(address) : false}
```
Comment (Creepypoke) [inline src/components/AssetItem/AssetItem.js]: What about `checkETH(address) || isActive` ?

**Comment by stcherenkov** [comment]
I suggest we also add link from balance to assets list (`/digital-assets/grid`)

**Comment by stcherenkov** [comment]
This PR actually consists of two, right? JWWEB-399 and displaying ETH balance in MenuPanel?

File: src/components/AssetItem/AssetItem.js (hunk 1 above)
Comment (Creepypoke) [reply src/components/AssetItem/AssetItem.js]: I think we need that Ethereum assets will be active all time, or not?

Comment (IViol) [reply src/components/AssetItem/AssetItem.js]: we should lock Ethereum to disable when it is already active

**Pull request merged**